docs/Authorization.md

```diff
@@ -59,6 +59,41 @@ $ bin/pulsar-admin namespaces grant-permission \
 After this, clients identifying with the role `my-client-role`, will be
 able to use topics in the specified namespace.
 
+When `authorizationAllowWildcardsMatching` is `true` in `broker.conf`, wildcard matching is also allowed.
```

Comment:
Should we rephrase the sentence?
Wildcard authorization can be performed when `authorizationAllowWildcardsMatching` is set to `true` in `broker.conf`.